OFFSPEC_runs_46992-4/README.md

```diff
@@ -1,4 +1,6 @@
-# Neutron Reflectivity Reference Data Set: OFFSPEC
+# Neutron Reflectivity Reference Data Sets
 
-This data set represents a published study of the determination of a thin-film structure using Neutron Reflectivity: *ACS Appl. Mater. Interfaces* 2019, 11, 14, 13803–13811 (https://doi.org/10.1021/acsami.9b02700). The raw data is available at https://data.isis.stfc.ac.uk/doi/INVESTIGATION/84794745/. The [raw data nexus file](https://github.com/DAaaS-reference-data/Reflectivity/blob/main/OFFSPEC_runs_46992-4/Runs_46992_46993_46994_polymer_on_Si.nxs), the [reduced data](https://github.com/DAaaS-reference-data/Reflectivity/blob/main/OFFSPEC_runs_46992-4/Runs_46992_46993_46994_polymer_on_Si.dat) and the [reduction script](https://github.com/DAaaS-reference-data/Reflectivity/blob/main/OFFSPEC_runs_46992-4/RB1720415_run_46992.py) to obtain one from the other are all included in this directory.
+This repository contains a Neutron Reflectivity data set taken on the [OFFSPEC](https://www.isis.stfc.ac.uk/Pages/offspec.aspx) instrument at the [ISIS Neutron and Muon Source](https://www.isis.stfc.ac.uk/) (UK). The OFFSPEC data set represents a published study of the determination of a thin-film structure using Neutron Reflectivity: *ACS Appl. Mater. Interfaces* 2019, 11, 14, 13803–13811 (https://doi.org/10.1021/acsami.9b02700). The purpose of this data sets is to act as reference data set to test and validate Data Analysis as a Service (DAaaS) platforms, for example as part of the [EU ExPaNDS project](https://expands.eu/).
```

Comment:
I suggest rephrasing this Readme a bit to better merge the two paragraphs (which originally came from two different Readme's) and to remove the sentence about the purpose of the data set as that is already included in the Readme for the project.